apps/Caskaydia Cove NF/install

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+mkdir cascadia
+cd cascadia
+wget https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.1/CascadiaCode.zip || error "failed to download font"
+unzip CascadiaCode.zip || error "failed to extract zip"
+mkdir ~/.fonts
+mkdir ~/.fonts/Caskaydia
+rm -- *Propo*
+rm -- *Mono*
+rm CascadiaCode.zip
+mv * ~/.fonts/Caskaydia/
```

Comment:
this only applies to the current user. when possible it is best to save files to the global directories so that they apply for all users. that would be `/usr/share/fonts` (for system installed fonts) and `/usr/local/share/fonts` for user installed fonts (the latter of which you should use)

also, why are specific fonts being removed after downloading? are they not desireable for some reason? maybe you should take that up with upstream ryanoasis if you do not like them.